tests/ops/test_qubit_ops.py

```diff
@@ -1057,5 +1057,5 @@ def test_decomposition(self):
         decomp = qml.DiagonalQubitUnitary.decomposition(D, [0, 1, 2])
 
         assert decomp[0].name == "QubitUnitary"
-        assert decomp[0].wires == [0, 1, 2]  #Wires([0, 1, 2])
```

Comment:
I admire your patience on this PR :smile:

Reply:
I admire your patience making it up to here in the review! ;)